### DISCUSSION TITLE
Why does "account posix set_password" prompt for "unit (sudo) password?

### DISCUSSION THREAD

**Discussion by yaleman** [kanidm/kanidm#587] [Q&A] [answered]
Per title 😄

**Comment by Firstyear** [comment, marked as answer]
Probably a typo. Should say "unix" instead of "unit". But the intent is the posix pw is really only for sudo as ssh key is the prefered method to access a system. I'm not really targetting laptop/desktop envs with Kani here, just linux servers.

**Reply by yaleman** [reply]
Thanks for the confirmation 😄
